src/presentational-components/shared/toolbar-addPermission-btn.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { routes as paths } from '../../../package.json';
+import { Button } from '@patternfly/react-core'; 
+import { Link } from 'react-router-dom';
+
+const toolbarAddPermissionBtn = () => {userEntitlements?.cost_management ? (
```

Comment:
this function is missing the return statement (or remove the `{` `}` block). Also as a react component, the function name should be capitalized.